Resources/doc/cookbook/recipe_persisting_filters.rst

```diff
@@ -0,0 +1,78 @@
+Persisting Filters
+==================
+
+Persisting filters allow your application to save the filters the authenticated user has submitted.
+Then the saved filters will be reused if the page is displayed again.
+
+
+Enable Filters Persistence
+--------------------------
+
+By default, filters persistence is disabled.
+You can enable it in your ``sonata_admin`` configuration :
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
+        sonata_admin:
+            persist_filters: true
+
+
+Choose the persistence strategy
+-------------------------------
+
+When you enable the filters persistence by setting ``persist_filters`` to ``true``.
+SonataAdmin will use the default filter persister : ``Sonata\AdminBundle\Filter\Persister\SessionFilterPersister`` (which is, by now, the only one provided).
+
+You can implement your own filter persister by creating a new class that implement the ``Sonata\AdminBundle\Filter\Persister\FilterPersisterInterface`` interface and registering it as a service.
```

Comment:
- "implements" with an "s" please
- Also, please comply with the sf docs standards as much as possible:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)